### ISSUE TITLE
Setting frequency via Task Scheduler

### ISSUE THREAD

**Issue by jdlarrimo** [PrimeO7/How-to-undervolt-AMD-RYZEN-5800X3D-Guide-with-PBO2-Tuner#18]
Good afternoon, and thank you for developing this tool for people to finally tame the 5800x3D temperature-wise.

Out of curiosity, is there any way to set the frequency via the task scheduler? I would love to set it at 4300 mHz at boot due to constraints in my PC not allowing for larger coolers.

**Comment by daigoNorr** [comment]
I'm also looking for a way to do this and a temp limit, would be useful. Thanks.

**Comment by jdlarrimo** [comment]
By checking out previous issues, I was able to find [anibalpedraza](https://github.com/anibalpedraza) who mentioned this:

> You have to add an extra 0 to the parameters.
> 
> The program, when providing more than 8 arguments, expects exactly 4 more arguments: PPT, TDC, EDC and Max Frequency. > Putting a 0 in the last one keeps the default value.

By taking this into consideration, I was able to get the task scheduler to set the frequency by including the following arguments for my setup:
"-25 -25 -25 -25 -25 -25 -25 -25 95 60 90 4300"
The last four are the arguments for the PPT, TDC, EDC, and the frequency target. After applying this and running the task once again, I reopened PBO2 Tuner to see that it had, in fact, adjusted the value correctly. Hope this helps those in the future!
